index.bs

```diff
@@ -583,6 +583,22 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     [=Registration=] and [=Authentication=] are ceremonies, and an [=authorization gesture=] is often a component of
     those [=ceremonies=].
 
+: <dfn>Client</dfn>
+: <dfn>[WAC]</dfn>
+:: Also referred to herein as simply a [=client=]. See also [=Conforming User Agent=]. A [=[WAC]=] is an intermediary entity typically implemented in the user agent (in whole, or in part). Conceptually, it underlies the [=Web Authentication API=] and embodies the implementation of the {{PublicKeyCredential/[[Create]](origin, options, sameOriginWithAncestors)}} and {{PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)}} [=internal methods=]. It is responsible for both marshalling the inputs for the underlying [=authenticator operations=], and for returning the results of the latter operations to the [=Web Authentication API=]'s callers.
+
+    The [=[WAC]=] runs on, and is distinct from, a [=[WAC] Device=].
+
+: <dfn>Client Device</dfn>
+: <dfn>[WAC] Device</dfn>
+:: The hardware device on which the [=[WAC]=] runs, for example a smartphone, a laptop computer or a desktop computer, and the
+    operating system running on that hardware.
+
+    The distinction between this and the [=client=] is that one [=client device=] MAY support running multiple [=clients=], i.e., browser
+    implementations, which all have access to the same [=authenticators=] available on that [=client device=]; and that [=platform
+    authenticators=] are bound to a [=[WAC] Device=] rather than a [=[WAC]=]. A [=client device=] and a [=client=] together make
+    up a [=client platform=].
```

Comment:
Perhaps we ought to polish this last parag a tad:

The distinctions between a [=[WAC] device=] and a [=client=] are:
    * a single [=client device=] MAY support running multiple [=clients=], i.e., browser implementations, which all have access to the same [=authenticators=] available on that [=client device=], and 
    * [=platform authenticators=] are bound to a [=client device=] rather than a [=[WAC]=]. A [=client device=] and a [=client=] together constitute a [=client platform=].

Reply:
Let's do that in a separate PR?

Reply:
Opened new issue #1216